santa.js

```diff
@@ -1,5 +1,23 @@
 function deliverPresents(houses) {
-  console.log('Delivering presents to ' + houses[0])
+  //Base case
+  //If we have a single house, deliver the presents and print delivered
+  if (houses.length === 0) return
+
+  if (houses.length === 1) {
+    console.log('Delivering presents to ' + houses[0])
+  } else {
+    //Break the Problem down
```

Comment:
While this comments are super helpful to us while solving the problem, we wouldn't want them to live in the code forever. Remember to always delete these notes to yourself once your code is working and you are ready to submit.